Fix creating core for a small number of levels

KaterynaSloboda opened the pull request:
BACKGROUND
- With a small number of levels, there was an error in the construction of the elevator

DESCRIPTION
- Error replaced with a warning for less than 2 levels and fixed an elevator creation error if there are at least 2 levels

Building Blocks contains many functions that are used in Hypar examples and in the Hypar tour. Please ensure that that following are true before granting this PR.

- [ ] The function has been published with staging, (but not released, until this PR is approved).
- [ ] The function has been tested against the tour. If the function is not used in the tour, mark this as not applicable (NA).
- [ ] Strings in `hypar.json` have been translated to supported languages in the `messages` field.
- [ ] Any new project added to this repository has been added to BuildingBlocks.sln.

anthonie-kramer reviewed (approved):
Is this correct? I am still getting a `Lift Shaft` for a 2 level building:
![image.png](https://files.reviewableusercontent.io/images/reviewable/8202304/jvMy50fKJaK9EPelXAW9pt9z/image.png)

I would prefer to have these elements work for all levels, rather than have a hard-coded limit

> Reviewed 9 of 12 files at r1, 3 of 3 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/hypar-io/BuildingBlocks/137#-:-NY3rDdM6iD1kEEDwH11:b-n47pvd)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @serenayl)



<!-- Sent from Reviewable.io -->

anthonie-kramer reviewed (changes requested):
I meant to block, I think we need a different fix

> *[Reviewable](https://reviewable.io/reviews/hypar-io/BuildingBlocks/137#-:-NY3r_rTEtpZr4AbEuY1:b-170mlz)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @serenayl)



<!-- Sent from Reviewable.io -->

KaterynaSloboda commented:
> I meant to block, I think we need a different fix
> 
> > _[Reviewable](https://reviewable.io/reviews/hypar-io/BuildingBlocks/137#-:-NY3r_rTEtpZr4AbEuY1:b-170mlz)_ status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @serenayl)

Changed: added additional checks to create all possible elements for a specific (even small) number of floors instead of a warning
